### ISSUE TITLE
Accuracy on flying chairs

### ISSUE THREAD

**Issue by bryanyzhu** [ClementPinard/FlowNetPytorch#10]
Hi, there. Thanks for the great code. I was trying to play with it on flying chair dataset. But the EPE in the end is around 5.0, which is far from 2.7 as stated in the original paper. So did you try to train the code from scratch and what performance do you have? Maybe I did something wrong. Thanks a lot.

**Comment by ClementPinard** [comment]
Hello, I have not ran thorough test of this implementation, apart from checking it actually converges, there may be some hyperparameters that are still needed to improve.

Be aware that EPE measured is not the real EPE, because here we compare our output to a downsampled Ground truth, wherehas we should rather upsample our output and compare it to the full resolution GT.

Also I realized a while ago that the original caffe code had color warping augmentation, which I don't have implemented.

If I had to do it again, there's a lot of thing I would change (and I did for my other pytorch project) but I don't have much time for the moment. If you find a way to improve performance to match original paper, I'd be glad to merge it ! ;)

**Comment by bryanyzhu** [comment]
Thanks for your reply. Yes, I compare it to the full resolution GT. I will try to see if i match the performance.

**Comment by isabellakaisen** [comment]
@bryanyzhu Hey,
How many epochs does it take to converge to an EPE of 5? Also, were you able to match the peformance after comparing to full resolution GT? Thanks.

**Comment by bryanyzhu** [comment]
@isabellakaisen Hi, I do 180 epochs, which is about 600,000 iterations. 

EPE of 5 (actually 5.3) is the performance of comparing to full resolution GT. Maybe there still remain some subtle problems in the code, I will dig into it.

**Comment by ClementPinard** [comment]
Glad to see it got some interest for you ! I'm planning on a cleaner, simpler implementation,we'll figure out how to reach 2.7 EPE more easily !

**Comment by bryanyzhu** [comment]
Sounds great, thanks for your effort.

**Comment by Yuliang-Zou** [comment]
Hi @ClementPinard , have you tested the performance of the converted weight from Caffe? My EPE is around 5.
